exercises/D-logical-operators/exercise.js

```diff
@@ -8,12 +8,15 @@
 // Do not change these two statement
 var htmlLevel = 8;
 var cssLevel = 4;
+var htmlLevelAbove5 = htmlLevel > 5;
+var cssLevelAbove5 = cssLevel > 5;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
 var htmlLevelAbove5;
 var cssLevelAbove5;
-
+var cssAndHtmlAbove5 = htmlLevelAbove5 && cssLevelAbove5;
```

Comment:
By using var and not using the pre-prepared variable declarations below, you have allowed your variable to be re-declared.